### ISSUE TITLE
Cannot disable root login

### ISSUE THREAD

**Issue by scottbomb** [phpmyadmin/phpmyadmin#18874]
### Describe the bug

I've seen instructions likes those given here to disable root login from the web page: https://tecadmin.net/how-to-disable-root-login-in-phpmyadmin/

Unfortunately, this doesn't seem to work on an ubuntu server 22.04 running on AWS. 

### To Reproduce

Install LAMP stack along with phpmyadmin. 
Create a login user in addition to root that has root privs
log out 
modify /usr/share/phpmyadmin/config.sample.inc.php

by appending: 

$cfg['Servers'][$i]['auth_type'] = 'cookie';
$cfg['Servers'][$i]['AllowRoot'] = FALSE;

systemctl daemon-reload
systemctl restart apache2

Go to login page and attempt to log in as root. You should not be able to but I can every time.

I had to destroy my instance and spin up a new one. Reinstalled everything and it happened again. Using same config I've used for years but this is the first time I've tried disabling web login for root (mysql root).

### Expected behavior

With the added lines above, one should not be able to login to phpmyadmin as root.

### Server configuration

- Operating system: ubuntu 22.04
- Web server: apache2 Server version: Apache/2.4.52 (Ubuntu)
- Database version: mysql  Ver 8.0.35-0ubuntu0.22.04.1 for Linux on x86_64 ((Ubuntu))
- PHP version: PHP 8.1.2-1ubuntu2.14 
- phpMyAdmin version: 5.1.1

### Client configuration

- Browser: brave/chrome
- Operating system: kubuntu 22.04

**Comment by ibennetch** [comment]
That path looks wrong. If you're using a packaged version of phpMyAdmin, the file is usually either `/etc/phpmyadmin/config.inc.php` or something else in `/etc/phpmyadmin` that is included by the package manager. If you're using the standard download that you've extracted yourself from phpmyadmin.net, then the file name is `config.inc.php` in the top level/root of your downloaded folder.

phpMyAdmin does not read the sample file, so modifying it will have no effect, and in most cases for a manual install you wouldn't put phpMyAdmin in `/usr/share/phpmyadmin/` or for a packaged version you'd modify the file in `/etc/` so even the full path in addition to the filename seems wrong here.

**Comment by ibennetch** [comment]
I've just tested again that the directive does work as expected, so it definitely looks like the file you're editing just isn't part of the configuration file loaded by phpMyAdmin.

If you continue to have trouble, let us know where exactly you've put your downloaded phpMyAdmin folder and if you've created any other configuration file (config.inc.php) for it previously.

**Comment by scottbomb** [comment]
Interesting. It's the vanilla phpmyadmin as downloaded from the Ubuntu repos. I haven't created any custom config files for it. The file /etc/phpmyadmin/config.inc.php does exist but I didn't put it there. I'll keep poking around with the config.

**Comment by ibennetch** [comment]
Yeah, if you're using the one from the Ubuntu repo then it's either the /etc/phpmyadmin/config.inc.php or add your own snippet to /etc/phpmyadmin/conf.d/ (this is the Debian/Ubuntu style of adding custom configuration without modifying the distributed config.inc.php file, but is not required).

**Comment by Jemt** [comment]
I can confirm this bug on Debian 12. I have made sure I'm modifying the correct config file: `/etc/phpmyadmin/config.inc.php`.

I have `$cfg['Servers'][$i]['allowroot'] = false;`, yet I'm still able to log in as root. This is absolutely terrifying in a production environment where root is suddenly allowed to log in after upgrading and using an existing configuration file.

The following can be used to prevent root logins instead:
```
$cfg['Servers'][$i]['AllowDeny']['order'] = 'allow,deny';
$cfg['Servers'][$i]['AllowDeny']['rules'] = array('allow % from all', 'deny root from all');

How has this bug been allowed to exist for more than 8 months ?

**Comment by liviuconcioiu** [comment]
This happens because `AllowRoot` is case-sensitive. Just checked and I can reproduce it too.

It should be `$cfg['Servers'][$i]['AllowRoot'] = false;` and **NOT** `$cfg['Servers'][$i]['allowroot'] = false;`

@MauricioFauth, @williamdes thoughts?

**Comment by MauricioFauth** [comment]
> This happens because `AllowRoot` is case-sensitive. Just checked and I can reproduce it too.
> 
> It should be `$cfg['Servers'][$i]['AllowRoot'] = false;` and **NOT** `$cfg['Servers'][$i]['allowroot'] = false;`

Yes, **all** config directives are case-sensitive.

**Comment by williamdes** [comment]
From what I understand there is no action to be taken, let me know if there is a mis-understanding.
